flow_action_components/CollectionProcessors/force-app/main/default/classes/AddOrInsertRecord.cls

```diff
@@ -1,5 +1,5 @@
 global with sharing class AddOrInsertRecord {
-    @InvocableMethod(label='Add or Insert Record [USF Collection Processor]' category='Util' iconName='resource:CollectionProcessorsSVG:colproc')
```

Comment:
It looks like this PR doesn't have the recent PR with thhe category and icon names. @AndyHaas , can you do a git pull and then update your PR so it's no longer trying to remove these lines?

Reply:
My last push fixed this issue.